contribs/Makefile

```diff
@@ -25,15 +27,29 @@ GOTEST_FLAGS ?= -v -p 1 -timeout=30m
 ########################################
 # Dev tools
 .PHONY: install
-install: install.gnomd install.gnodev install.gnofaucet
+install:
+	@set -e; for program in $(programs); do ( set -e; \
+	  echo "[+] make -C $$program install"; \
+	  $(MAKE) -C $$program install; \
+	); done
 
```

Comment:
I don't think installing all programs is a "typical" workflow; I see `contribs` more as a list of optional tools that a user should understand according to what they need.

I think it's fine for test and lint, although I kind of think at this point that the two commands could be simple calls to `go test` and `golangci-lint` from this directory with `./...`.

One thing: if we're removing `install.gnodev` from this makefile, the top-level makefile also needs to be updated, as it has these lines:

```
.PHONY: install.gnodev
install.gnodev:
	$(MAKE) --no-print-directory -C ./contribs install.gnodev
```

Reply:
It's not about encouraging it, but enabling validation of each folder's installation status. This is akin to running "make test" at the top-level to ensure everything is working well, without focusing on specific changes. 

My PR requires every subfolder in contribs/ to have its own makefile with the install command, which is the preferred method for installation. Use `cd contribs/mytool; make install`.

I will update the top-level makefile accordingly and rename `make install` to `make install_all`.